fetcher/fetcher_multiple.go

```diff
@@ -0,0 +1,104 @@
+package fetcher
+
+import (
+	"io"
+	"sync"
+)
+
+// interface validation
+var _ Interface = &Multiple{}
+
+// Multiple uses multiple fetcher.Interface.
+//
+// e.g.) fetching from both of local file and GitHub.
+//   overseer.Config{
+//   	Fetcher: &Multiple{
+//   		List: []fetcher.Interface{
+//   			&fetcher.File{
+//   				Path: "/path/to/my_app",
+//   			},
+//   			&fetcher.Github{
+//   				User: "jpillora",
+//   				Repo: "overseer",
+//   			},
+//   	},
+//   }
+//
+type Multiple struct {
+	// List for setting multiple fetchers.
+	List []Interface
+
+	result          chan fetchResult
+	runningStatusMu sync.RWMutex
+	runningStatus   map[int]bool
```

Comment:
`result` is channel type because used for retrieving the single fastest results from fetchers.
To avoid duplicate goroutine running,  `runningStatus` is used for managing fetchers' running state.